File: Api/Api/Core/Models/Policy.cs
using System;

namespace Api.Core.Models
{
	public class Policy
	{
		public int Id { get; set; }
		public string Name { get; set; }
		public string Description { get; set; }
		public DateTime StartValidity { get; set; }
		public int CoverPeriod { get; set; }
		public decimal Price { get; set; }
		public Coverage Coverage { get; set; }
		public byte CoverageId { get; set; }
		public Risk Risk { get; set; }
		public byte RiskId { get; set; }


	}
}

File: Client/src/app/admin/Policy/policy.ts
export class Policy {
  constructor(
    public id: number,
    public name: string,
    public description: string,
    public startValidity: string,
    public coverPeriod: string,
    public price: string,
    public coverageId: string,
    public riskId: string,
  ) {}
}
